examples/ReadSensorsImperial/ReadSensorsImperial.ino

```diff
@@ -0,0 +1,68 @@
+/*
+  MKR ENV Shield - Read Sensors Imperial
+
+  This example reads the sensors on-board the MKR ENV shield
+  and prints them in imeprial units to the Serial Monitor once a second.
+
+  The circuit:
+  - Arduino MKR board
+  - Arduino MKR ENV Shield attached
+
+  This example code is in the public domain.
+*/
+
+#include <Arduino_MKRENV.h>
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial);
+
+  if (!ENV.begin()) {
+    Serial.println("Failed to initialize MKR ENV shield!");
+    while (1);
+  }
+}
+
+void loop() {
+  // Seting to FAHRENHEIT and FOOTCANDLE readTemperature and readIlluminance
+  // allow to reads in imperial units the temperature and illuminance sensor 
+  float temperature = ENV.readTemperature(FAHRENHEIT);
+  float humidity    = ENV.readHumidity();
+  float pressure    = ENV.readPressure();
```

Comment:
I think we are also missing the imperial units for pressure? Is it PSI?

Reply:
Here they use inches? https://www.wunderground.com/weather/us/ny/manhattan/40.75%2C-74.00

Reply:
`the pound-force per square inch (psi) is the traditional unit of pressure in the imperial and US customary ` then i add the PSI support

Reply:
done